account_fiscal_year_closing/views/account_fiscalyear_closing_views.xml

```diff
@@ -0,0 +1,196 @@
+                    />
+                    <button name="button_recalculate"
+                            states="calculated"
+                            type="object"
+                            string="Recalculate"
+                    />
+                    <button name="button_post"
+                            type="object"
+                            string="Confirm and post moves"
+                            class="oe_highlight"
+                            attrs="{'invisible': ['|', ('move_config_ids', '=', []), ('state', 'not in', ['calculated'])]}"
+                    />
+                    <button name="button_recover"
+                            states="cancelled"
+                            type="object"
+                            string="Draft"
+                    />
+                    <button name="button_cancel"
+                            states="calculated,posted"
+                            type="object"
+                            string="Cancel"
+                            confirm="This will remove all the closing moves. Are you sure you want to continue?"
+                    />
+                    <field name="state"
+                           widget="statusbar"
+                           statusbar_visible="draft,calculated,posted,cancelled"
+                           statusbar_colors="{'cancelled': 'red', 'posted': 'blue'}"
+                    />
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button class="oe_stat_button"
+                                name="button_open_moves"
+                                states="calculated,posted"
+                                type="object"
+                                string="Show moves"
+                                icon="fa-list"
+                        />
+                        <button class="oe_stat_button"
+                                name="button_open_move_lines"
+                                states="calculated,posted"
+                                type="object"
+                                string="Show move lines"
+                                icon="fa-list"
+                        />
+                    </div>
+                    <h1>
+                        <label string="Fiscal closing"/>
+                        <field name="name" class="oe_inline"/>
+                    </h1>
+                    <group>
+                        <group>
+                            <field name="chart_template_id" invisible="1"/>
+                            <field name="stored_template_id" invisible="1"/>
+                            <field name="is_new_template" invisible="1"/>
+                            <label for="closing_template_id"/>
+                            <div>
+                                <field name="closing_template_id" class="oe_inline"/>
+                                <button name="action_load_template"
```

Comment:
I found issues here. When you have to change the template for some reason, then you do not have an option to reload anymore, and it is quite a mess to work around it. In my opinion as long as the account.fiscalyear.closing is in status 'draft' the user should be able to press the button "Load", and this would remove the previous record and load new ones.

Reply:
That's not really true. Try again and see that you can change it whenever you change the template to use. If you have messed up the default configuration, then just switch to empty template, load, and switch back to the other template.

You shouldn't change the template too often, and this way you prevent an accidental reload, overwriting your changes.